controls/user_rights.rb

```diff
@@ -1,13 +1,11 @@
-# encoding: utf-8
-
 title 'User Rights Assignment'
 
 control 'cis-access-cred-manager-2.2.1' do
   impact 0.7
   title '2.2.1 Set Access Credential Manager as a trusted caller to No One'
   desc 'Set Access Credential Manager as a trusted caller to No One'
   describe security_policy do
-    its('SeTrustedCredManAccessPrivilege') { should eq ['S-1-0-0'] }
```

Comment:
@atomic111 Whay have you changed this? Have we used the wrong values before?

Reply:
yes this is the wrong value, because in the CIS language "No one" means a blank value